libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py

```diff
@@ -2128,28 +2130,29 @@ def process_line(line):
         env_vars[a[1]] = ' '.join(a[2:])
 
 
-def process_file(filename):
+def process_file(fname,extra_hwdef):
```

Comment:
I think it would be better to leave process_file() taking one file and calling it twice